src/common/imageio_jpeg.c

```diff
@@ -725,8 +725,7 @@ dt_imageio_retval_t dt_imageio_open_jpeg(dt_image_t *img, const char *filename,
 {
   const char *ext = filename + strlen(filename);
   while(*ext != '.' && ext > filename) ext--;
-  if(strncmp(ext, ".jpg", 4) && strncmp(ext, ".JPG", 4) && strncmp(ext, ".jpeg", 5)
-     && strncmp(ext, ".JPEG", 5))
+  if(g_ascii_strcasecmp(ext, ".jpg") && g_ascii_strcasecmp(ext, ".jpeg") && g_ascii_strcasecmp(ext, ".jfif"))
```

Comment:
Maybe add a comment that JFIF stands for "JPEG File Interchange Format" and is a plain jpeg format with some mandatory metadata?

Reply:
OK added